Ego4D-Future-Hand-Prediction/slowfast/datasets/ego4dhand.py

```diff
@@ -80,7 +80,7 @@ def _construct_loader(self):
         Construct the video loader.
         """
         path_to_file = os.path.join(
-            self.cfg.DATA.PATH_TO_DATA_DIR, "{}.json".format(self.mode)
+            self.cfg.DATA.PATH_TO_DATA_DIR, "fho_hands_{}.json".format(self.mode)
```

Comment:
This won't work for trainval or test (i.e. test_uannotated) will it?

Reply:
should work for fho_hand_{train, val} but not for the unannotated test now because it has a different structure and still needs the information of frames. Please refer to our slack channel for a more detailed description.